src/types/postType.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './commentType';
+import { User } from './userType';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  user?: User | null;
+  comments: Comment[];
+}
```

Comment:
it might be better to create an additional interface that would extends the post?

Reply:
In my opinion, my code more readable

Reply:
@Jamal-Amoian Your original post doesn't have fields `user` or `comments`. @Anastasiia-Khmelovska is right